llms/mlx_lm/lora.py

```diff
@@ -194,6 +194,17 @@ def load_dataset(args):
     return train, valid, test
 
 
+def print_trainable_parameters(model):
+    total_p = sum(v.size for _, v in tree_flatten(model.parameters())) / 10**6
+    trainable_p = (
+        sum(v.size for _, v in tree_flatten(model.trainable_parameters())) / 10**6
+    )
```

Comment:
Btw this is wrong for QLoRA since `v.size` does not correspond to the number of parameters.. I have been meaning to fix it.